{"name": "Проверка успешной аутентификации пользователя при передаче валидных кредов созданного аккаунта", "status": "passed", "start": 1729329582015, "stop": 1729329582496, "uuid": "58aa6788-92d9-4180-bd20-c060c4882cf9", "historyId": "504439d05b553f670ba5f42e19a4a3b4", "testCaseId": "504439d05b553f670ba5f42e19a4a3b4", "fullName": "tests.test_authentication.TestAuthentication#test_auth_existing_account_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_authentication"}, {"name": "subSuite", "value": "TestAuthentication"}, {"name": "host", "value": "DESKTOP-I4FH3MC"}, {"name": "thread", "value": "108300-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_authentication"}]}